lib/components-schema-v1_0_x.ts

```diff
@@ -5,6 +5,302 @@
+                        type: 'boolean',
+                        description: 'Enable opacity setting',
+                    },
+                },
+            },
+            {
+                additionalProperties: false,
+                required: ['type'],
+                properties: {
+                    type: {
+                        enum: ['image-editor'],
+                        description: 'Image editor field property',
+                    },
+                    focuspoint: {
+                        type: 'boolean',
+                        description: 'Enable focuspoint feature',
+                    },
+                },
+            },
+            {
+                additionalProperties: false,
+                required: ['type'],
+                properties: {
+                    type: {
+                        enum: ['drop-capital'],
+                        description: 'Drop capital field property',
+                    },
+                    charactersMinimum: {
+                        type: 'number',
+                        description: 'Minimum value of characters number',
+                    },
+                    charactersDefault: {
+                        type: 'number',
+                        description: 'Default value of characters number',
+                    },
+                    charactersMaximum: {
+                        type: 'number',
+                        description: 'Maximum value of characters number',
+                    },
+                    linesMinimum: {
+                        type: 'number',
+                        description: 'Minimum value of lines number',
+                    },
+                    linesDefault: {
+                        type: 'number',
+                        description: 'Default value of lines number',
+                    },
+                    linesMaximum: {
+                        type: 'number',
+                        description: 'Maximum value of lines number',
+                    },
+                },
+            },
+            {
+                additionalProperties: false,
+                required: ['type'],
+                properties: {
+                    type: {
+                        enum: ['media-properties'],
+                        description: 'Enables media properties field property',
+                    },
+                },
+            },
+            {
+                additionalProperties: false,
+                required: ['type'],
+                properties: {
+                    type: {
+                        enum: ['fitting'],
+                        description: 'Enables fitting option for an image directive',
+                    },
+                },
+            },
+            {
+                additionalProperties: false,
+                required: ['type'],
+                properties: {
+                    type: {
+                        enum: ['slides'],
```

Comment:
Seems this control type requires additional checkings in the validator like
- directive key should be valid
- works with doc-slideshow only
- other?
Can you please add a bullet to my current ticket?

Reply:
I will add it to the current pull request after you merge your ticket to master.